contrib/sarplus/scala/python/setup.py

```diff
@@ -1,8 +1,13 @@
 from distutils.core import setup
+import os
+
+version = os.getenv("VERSION")
+if version is None:
+    version = "0.5.0"
 
 setup(
     name="pysarplus_dummy",
-    version="0.2",
```

Comment:
Shouldn't we go from version 0.2 to 0.3 (instead of 0.5)?

Reply:
To avoid conflicting with the internal usage of pysarplus where the latest version is 0.4.10, I think 0.5.0 would be better.